types/react-virtual-keyboard/index.d.ts

```diff
@@ -0,0 +1,31 @@
+// Type definitions for react-virtual-keyboard 1.0
+// Project: https://www.npmjs.com/package/react-virtual-keyboard
+// Definitions by: Bogdan Surai <https://github.com/bsurai>
```

Comment:
What I meant is that npm package link doesnt have enough information to see if you are declaring correct types. It points to github.com/gaearon/react-hot-boilerplate as source and that seems to be incorrect?